lightkurve/injection.py

```diff
@@ -0,0 +1,288 @@
+"""Defines UniformDistribution, GaussianDistribution, TransitModel, and SupernovaModel"""
+
+import numpy as np
+from lightkurve import LightCurve
+import matplotlib.pyplot as plt
+from scipy.stats import norm
+
+class UniformDistribution(object):
+    """
+    Implements a class for choosing a value from a uniform distribution.
+
+    Attributes
+    ----------
+    lb : float
+        Lower bound of distribution
+    ub : float
+        Upper bound of distribution
+    """
+    def __init__(self, lb, ub):
+        self.lb = lb
+        self.ub = ub
+
+    def __repr__(self):
+        return 'UniformDistribution(lb={},ub={})'.format(self.lb,
+                                         self.ub)
+
+    def sample(self):
+        """Chooses values from uniform distribution.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        values : array-like
+            Returns array of randomly chosen values.
+        """
+        return np.random.uniform(self.lb, self.ub)
+
+    def plot(self):
+        """Plots specified distribution."""
+        t = np.arange(self.lb, self.ub, 0.01)
```

Comment:
So that we can see the edges of the uniform distribution, why don't you make the range something like

`np.linspace(self.lb*0.9, self.ub*1.1, 1000)`

(note `np.linspace` will allow you to fix it to 1000 points. A `np.arange(0, 1e10, 0.01)` will have 1e12 points and we probably don't need that much precision!